src/sass/components/_brands-slider.scss

```diff
@@ -0,0 +1,94 @@
+.brands-container {
+  position: relative;
+  padding: 20px 15px;
+  border-top: 1px solid #cccccc;
+  border-bottom: 1px solid #cccccc;
+  margin-bottom: 20px;
+}
+
+.slick-slide {
+  margin: 0px 16px;
+}
+
+.slick-slider {
+  position: relative;
+  display: block;
+  box-sizing: border-box;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  -webkit-touch-callout: none;
+  -khtml-user-select: none;
+  -ms-touch-action: pan-y;
+  touch-action: pan-y;
+  -webkit-tap-highlight-color: transparent;
+  padding: 0 30px;
+}
+
+.slick-list {
+  position: relative;
+  display: block;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+}
+
+.slick-list:focus {
+  outline: none;
+}
+
+.slick-slide {
+  display: none;
+  float: left;
+  height: 100%;
+  min-height: 1px;
+}
+
+[dir="rtl"] .slick-slide {
+  float: right;
+}
+
+.slick-slide img {
+  display: block;
+}
+
+.slick-initialized .slick-slide {
+  height: 92px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #cccccc;
+}
+
+.slick-arrow.slick-hidden {
+  display: none;
+}
+
+.slider-arrow {
+  position: absolute;
+  top: 20px;
+  height: 92px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 0;
+  width: 33px;
+  justify-content: center;
+  background: #2a2a2a;
+  color: white;
+  font-size: 13px;
+
+  &:hover {
+    @extend %animated-element;
+    background: $primary;
+    cursor: pointer;
+  }
+}
+
+.left {
+  left: 0;
+}
+
+.right {
+  right: 0;
+}
```

Comment:
Moim zdaniem na podglądzie wygląda to super. Dodałabym jeszcze dla całości jakiś margin lub padding od góry i od dołu, żeby ta sekcja nie była tak "ściśnięta".

Reply:
Dodałem większy margines na dole i na górze!

Reply:
Dominiku, i jeszcze jedna drobna uwaga. Widzę, że żeby wypozycjonować strzałki nadałeś `position: relative` aż trzem elementom. Wystarczyłoby pewnie zostawić to tylko u rodzica, czyli `brands-container`.